### PR TITLE
Elamin-JS exercises

### PR DESCRIPTION
**Pull request by aldeez988**
My JS exercises

### CODE REVIEW

File: week-1/L-functions-nested/exercise.js [hunk 1]
```diff
@@ -3,3 +3,29 @@ var mentor2 = "Irina";
...
+function printGreeting(name) {
+  console.log("Hello " + name.toUpperCase());
+}
+
+percentage(15, 8);
+printGreeting("Daniel");
+printGreeting("Irina");
+printGreeting("Mimi");
+printGreeting("Rob");
+printGreeting("Yohannes");
```
Comment (Atayo) [inline week-1/L-functions-nested/exercise.js]: Great use of functions, very easy to understand. Great work.

File: week-1/D-variables/exercise.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `greeting`
 
+var greeting = "Hello world";
+for(var i=1;i<=3;i++)
```
Comment (AlexanderBraithwaite) [inline week-1/D-variables/exercise.js]: Good use of the for loop

File: week-1/E-strings/exercise.js [hunk 3]
```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
-
+var message = "Hi I am Elamin";
+var messageType = typeof message ;
 console.log(message);
+console.log(`The type of the variable message ${messageType}`);
```
Comment (AlexanderBraithwaite) [inline week-1/E-strings/exercise.js]: Nice use of template literals 👍

File: week-1/F-strings-concatenation/exercise.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var greetingStart = "Hello, my name is ";
```
Comment (AlexanderBraithwaite) [inline week-1/F-strings-concatenation/exercise.js]: Good use of camelCase variable names

File: week-1/K-functions-parameters/exercise5.js [hunk 5]
```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting(name, age) {
+  return "Hello, my name is " + name + " and I'm " + age + " years old";
+}
 const greeting = createLongGreeting("Daniel", 30);
```
Comment (AlexanderBraithwaite) [inline week-1/K-functions-parameters/exercise5.js]: Good use of const, make sure you know the difference between const and var 👍

File: week-1/L-functions-nested/exercise.js (hunk 1 above)
Comment (AlexanderBraithwaite) [reply week-1/L-functions-nested/exercise.js]: Good work 👍

Comment (aldeez988) [reply week-1/L-functions-nested/exercise.js]: Thank you  @Atayo  and @AlexanderBraithwaite

File: week-1/K-functions-parameters/exercise5.js (hunk 5 above)
Comment (aldeez988) [reply week-1/K-functions-parameters/exercise5.js]: Thanks @AlexanderBraithwaite . Yes I will do  +1.

File: week-1/L-functions-nested/exercise.js (hunk 1 above)
Comment (ahmedahashim) [reply week-1/L-functions-nested/exercise.js]: Great work

**Pull request closed without merging**